tests/Fixture/ArticlesFixture.php

```diff
@@ -13,9 +13,10 @@ class ArticlesFixture extends TestFixture
      */
     public $fields = [
         'id' => ['type' => 'string', 'null' => false, 'length' => 36],
-        'title' => ['type' => 'string', 'null' => false, 'default' => null],
         'created' => ['type' => 'datetime', 'null' => true, 'default' => null],
         'modified' => ['type' => 'datetime', 'null' => true, 'default' => null],
+        'title' => ['type' => 'string', 'null' => false, 'default' => null],
```

Comment:
I think it would be worthwhile to at least recommend one way in `cakephp/docs` (and I don't care which way, it just makes it easier to work on cake apps in future). Reverted.